src/NovelBasicFillRect.cpp

```diff
@@ -35,14 +35,12 @@ void NovelBasicFillRect::configureBuffer() {
       bottomRight.getX(), bottomRight.getY(), 0.0f,
   };
 
-  GLfloat targetArray[_vertexBufferData.size()];

```

Comment:
This is depending on C99 behavior, which MSVC doesn't support.

However, directly using the underlying buffer is better anyways.